src/components/Header/Header.jsx

```diff
@@ -29,6 +33,10 @@ const Header = ({ companyName }) => {
       .catch((err) => console.log(err.response));
   }
 
+  const handleLogClick = () => {
```

Comment:
I would rename this to `onGotoDeliveryLogsPageRequested` as this is clearer in the intention vs handleLogClick